share/completions/arc.fish

```diff
@@ -0,0 +1,294 @@
+# fish completion for arc
+
+function __fish_arc_needs_command
+  set -l cmd (commandline -opc)
+  if [ (count $cmd) -eq 1 -a $cmd[1] = 'arc' ]
```

Comment:
You _really_ don't want this idiom. We removed it from the codebase (IIRC entirely) for a reason. This will break when you alias arc (because of the `$cmd[1] = 'arc'`, and this will break when you enter an option before a command (which arc might or might not support).

What you want is, depending on how arc works, do something like

```fish
for token in $cmd
    switch $token
        case '-*'
            continue
        case '*'
            return 1
     end
     return 0
```

This will assume the first non-option is a command.

If you want to be more precise (because e.g. arc can take arguments to options before the command), take a look at the new git.fish.